src/foam/u2/crunch/CapabilityStore.js

```diff
@@ -473,7 +475,22 @@ foam.CLASS({
             if ( ucj && ( ucj.status == this.CapabilityJunctionStatus.GRANTED
               || ucj.status == this.CapabilityJunctionStatus.PENDING ) ) return;
 
-            this.openWizard(cap, false);
+            let x = this.ctrl.__subContext__;
```

Comment:
This entire block of code is written inside `if ( sink.array.length == 1 )`

if it's convenient, can you invert it so it's like this?
```javascript
if ( ! sink.array.length == 1 ) return;
```
followed by the rest of the code

Reply:
@KernelDeimos I retested...

seems ‘data’ can not be put in that way when  a cap. set Granted due to validation issue. (such as inisialBusinessData)
But for General-admission which using its ‘of’ SubmitData it needs something below

var data = null;
if ( net.nanopay.crunch.onboardingModels.SubmitData === cap.of ) {
  data = net.nanopay.crunch.onboardingModels.SubmitData.create({
     submitted: true
   });
}